experiments/ClimaCore/sea-ice/slab_ice.jl

```diff
@@ -16,6 +16,9 @@ using OrdinaryDiffEq
 
 using Statistics
```

Comment:
We can delete this file in fact. I don't this it contains much more than we already have in the eisenman CI runs rn. 🧊